[V9] BugFix: ConfigCommand load only from application config

HamedDarragi opened the pull request:
ConfigCommand load only application config after splitting FileLoader in two loaders ( See https://github.com/concrete5/concrete5/commit/2514f1ac195ad62fff240f4cd8aa948a8faab7c8 )

aembler commented:
I'm not sure I understand what this pull request is trying to solve - I think the restructuring of the code is making this harder to read than it might be otherwise. What is this pull request addressing exactly?

HamedDarragi commented:
When you try to get config value using `concrete\bin\concrete5 c5:config get ` command you will always get null value if the item you're looking doesn't exist under `application/config` nor `application/config/generated_overrides`.

The previously used class `Concrete\Core\Config\FileLoader` was changed by Korvin Commit https://github.com/concrete5/concrete5/commit/2514f1ac195ad62fff240f4cd8aa948a8faab7c8#diff-4933532a35114103993424a03909a0e080f6535592b99a401652a159250b06c6L47 and removed core namespace from it and replaced the FileLoader by `Concrete\Core\Config\CompositeLoader` https://github.com/concrete5/concrete5/commit/2514f1ac195ad62fff240f4cd8aa948a8faab7c8#diff-9948fd21643c02c1fbe22560b99eaf8d9e02a7b8e831431e288aafa7bda0e4b7R26

So the only change i made is that i have replaced  `Concrete\Core\Config\FileLoader`  with `Concrete\Core\Config\LoaderInterface`.
https://github.com/concrete5/concrete5/pull/9103/files#diff-c9984014adcdc8a02bdfa79edc82e5baac0cc44bb840814962c1dfb66bf026e4L176

I accidentally committed my change with Code Format in same commit, i've already separated them in [V8 PR#9104](https://github.com/concrete5/concrete5/pull/9104/commits/1785c07763f12877a2b987a2dd8b8ff3cf87b67f)
